src/Nowruz/modules/general/components/cardRadioButton/cardRadioButtonIcon.tsx

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 import variables from 'src/components/_exports.module.scss';
+import { Icon, IconProps } from 'src/Nowruz/general/Icon';
 
 import css from './cardRadioButton.module.scss';
 
 interface CardRadioButtonIconProps {
   selected: boolean;
+  icon: IconProps;
 }
 
-export const CardRadioButtonIcon: React.FC<CardRadioButtonIconProps & { Component: React.ComponentType }> = ({
-  selected,
-  Component,
-}) => {
+export const CardRadioButtonIcon: React.FC<CardRadioButtonIconProps> = ({ selected, icon }) => {
   return (
     <div className={css.iconDiv}>
```

Comment:
I suggest sending an Icon as props cause you might also need some styles for the icon